api/k8s/v1/k8s.proto

```diff
@@ -369,6 +369,9 @@ message HPA {
 
   map<string, string> labels = 5;
   map<string, string> annotations = 6;
+
+  google.protobuf.Int32Value current_cpu_utilization_percentage = 7;
```

Comment:
why is the type `google.protobuf.Int32Value` rather than a regular `int32`

Reply:
to prevent it being defaulted to 0.  If it just an int32 instead of an Int32Value (which is a pointer), then that means we can't tell the difference between it actually being 0 and it just not being set, which would default to 0.  If its an Int32Value, we can have it be null in the case where its not set, and 0 when it is actually set.

If we can be sure that it will always be set, or if we don't mind it being 0 defaulted, then we can change it to an int32 rather than an Int32Value.